HomeWork_11/HW11.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+from time import perf_counter
+
+
+
+def decorator(func):
+    def timer():
+        start = perf_counter()
+        def inner():
+            return perf_counter() - start
+        return inner
+
+def counter(func):
+    count = 0
+    def inner(*args,**kwargs):
+        nonlocal count
+        count +=1
+        print(f"Функція {func.__name__} викликалась {count} разів")
+        return func(*args,**kwargs)
+
+
+@counter
+@decorator
+def add(a, b):
+    a = int(input("first digit "))
+    b = int(input("twice digit "))
+    return a + b
+
+
+
+
+# #-----------------------------------------
+def MyCustomException(Exception):
+    pass
+try:
```

Comment:
To raise an exception you should use `raise` command:
`raise MyCustomException("Custom exception is occurred)`